spec/requests/errors_spec.rb

```diff
@@ -61,15 +61,17 @@
     end
   end
 
-  it "forbids users without pre-release permission from accessing pre-release documents" do
-    pre_release_document_type = build(:document_type, pre_release: true)
-    edition = create(:edition, document_type: pre_release_document_type)
-    user = build(:user, permissions: %w(signin))
+  describe "ANY /pre-release-document-path" do
+    it "returns forbidden when the document type is pre-release" do
+      pre_release_document_type = build(:document_type, pre_release: true)
```

Comment:
Not blocking: A `pre_release` [trait](https://github.com/alphagov/content-publisher/blob/master/spec/factories/document_type_factory.rb#L41) was added to the document_type factory after this test was added. I think most of the other tests were updated to use this trait, but it looks like this test was missed. Could it be updated as well?

Reply:
Good spot! Also updated.